src/patternfly/components/Login/__login-main-footer.hbs

```diff
@@ -1,29 +1,29 @@
 {{#> login-main-footer}}
   {{#> login-main-footer-links}}
     {{#> login-main-footer-links-item}}
-      {{#> login-main-footer-links-item-link login-main-footer-links-item-link--attribute='aria-label="Log in with Google"'}}
-        <svg aria-hidden="true" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 488 512"><path d="M488 261.8C488 403.3 391.1 504 248 504 110.8 504 0 393.2 0 256S110.8 8 248 8c66.8 0 123 24.5 166.3 64.9l-67.5 64.9C258.5 52.6 94.3 116.6 94.3 256c0 86.5 69.1 156.6 153.7 156.6 98.2 0 135-70.4 140.8-106.9H248v-85.3h236.1c2.3 12.7 3.9 24.9 3.9 41.4z"/></svg>
-      {{/login-main-footer-links-item-link}}
+      {{#> button button--IsPlain=true button--aria-label="Log in with Google"}}
```

Comment:
Could be a followup, but with this update the icon color for the "log in with..." buttons no longer works in dark theme:

![Log in buttons with dark text in dark theme](https://github.com/patternfly/patternfly/assets/70952936/a2d672f7-602c-4617-b75e-f72f2f80ddc8)

Looks like it's because we're no longer targeting the `fill` property for the icon color here after removing the `login-main-footer-links-item-link` component + styling.

Reply:
nice! I meant to add `fill="currentColor"` to those SVGs. updated

https://github.com/patternfly/patternfly/pull/6328/commits/ab289581b94910efeeddcf72c25e7cd4b7165d6f

https://patternfly-pr-6328.surge.sh/components/login-page/html/basic/